### ISSUE TITLE
 Parameter "kibana_access: ro" not working for 6.1.0

### ISSUE THREAD

**Issue by sanacl** [sscarduzio/elasticsearch-readonlyrest-plugin#314]
Hi,
I'm testing the ES plugin with elasticsearch 6.1.0 and kibana and I'm not able to get the kibana_access parameter working for read only operations. 

According to the zip file, the version of the plugin is 1.16.15 with md5 93d5de0d2fe8315acbb2371d1f4bd9d6

This is the log I'm getting:

```
[2018-02-14T09:39:57,578][INFO ][t.b.r.a.ACL              ] FORBIDDEN by default req={ ID:554508840-63789460#84, TYP:GetMappingsRequest, CGR:N/A, USR:bitergia, BRS:false, ACT:indices:admin/mappings/get, OA:172.17.0.3, IDX:.kibana, MET:GET, PTH:/.kibana/_mappings, CNT:<N/A>, HDR:Authorization,Connection,Content-Length,Host, HIS:[Accept requests from users in group admin->[groups->false]], [RO for Kibana->[kibana_access->false, auth_key->true]] } 
```

And this is the setup I'm using:
```
readonlyrest:
    #optional
    response_if_req_forbidden: Sorry, your request is forbidden.

    access_control_rules:

    - name: Accept requests from users in group admin
      groups: ["admin"]

    - name: RO for Kibana
      hosts: [172.17.0.3]
      kibana_access: ro
      kibana_index: ".kibana"
      groups: ["readers"]

    users:
    - username: admin
      auth_key_unix: admin:$6$rounds=65535$d07dnv4N$QeErsDT9Mz.ZoEPXW3dwQGL7tzwRz.eOrTBepIwfGEwdUAYSy/NirGoOaNyPx8lqiR6DYRSsDzVvVbhP4Y9wf0 #admin:test
      groups: ["admin"]

    - username: bitergia
      auth_key: bitergia:bitergia # auth_key is good for testing but replace it with auth_key_sha1 for production!
      groups: ["readers"]
```

**Comment by sscarduzio** [comment]
Yes you are right. I added the action "indices:admin/mappings/get" in the white list.

**Comment by sscarduzio** [comment]
This fix is present in 1.16.16, which you can now [download from the website](https://readonlyrest.com/download.html)

**Comment by sanacl** [comment]
Thanks for the quick reply @sscarduzio :)
